msal4j-sdk/src/main/java/com/microsoft/aad/msal4j/AcquireTokenSilentSupplier.java

```diff
@@ -113,15 +119,15 @@ private boolean shouldRefresh(SilentParameters parameters, AuthenticationResult
 
         //If forceRefresh is true, no reason to check any other option
         if (parameters.forceRefresh()) {
-            setCacheTelemetry(CacheTelemetry.REFRESH_FORCE_REFRESH.telemetryValue);
+            setCacheTelemetry(CacheRefreshReason.FORCE_REFRESH);
             log.debug("Refreshing access token because forceRefresh parameter is true.");
             return true;
         }
 
         //If the request contains claims then the token should be refreshed, to ensure that the returned token has the correct claims
         //  Note: these are the types of claims found in (for example) a claims challenge, and do not include client capabilities
         if (parameters.claims() != null) {
-            setCacheTelemetry(CacheTelemetry.REFRESH_FORCE_REFRESH.telemetryValue);
+            setCacheTelemetry(CacheRefreshReason.FORCE_REFRESH);
```

Comment:
Can we have a separate entry for Claims please?

Reply:
Should there actually be a separate enum entry, or should it be like MSAL .NET and have the name reference both force refresh and claims? https://github.com/AzureAD/microsoft-authentication-library-for-dotnet/blob/7d17705533eab68d504ee2f179364922367dfba7/src/client/Microsoft.Identity.Client/Cache/CacheRefreshReason.cs#L18

It makes sense for them to be separate, however I think both of those reasons map to the same telemetry value.